elfo-network/src/discovery/mod.rs

```diff
@@ -550,6 +584,37 @@ async fn recv_regular<M: Message>(socket: &mut Socket, idle_timeout: Duration) -
     })
 }
 
+#[cfg_attr(test, derive(Debug))]
+struct Diff<T> {
```

Comment:
Let's move it to a dedicated module (`discovery/diff.rs`)

Reply:
Done